AppHandling/Run-AlValidation.ps1

```diff
@@ -295,8 +296,7 @@ else {
     $RemoveBcContainer = { Param([Hashtable]$parameters) Remove-BcContainer @parameters }
 }
 
-$vsixFile = DetermineVsixFile -vsixFile $vsixFile
-
+$vsixFile = GetAlLanguageExtension -vsixFile $vsixFile -extract;

```

Comment:
Extracting the vsixFile here will disable the GitHub cache - on GitHub hosted runners, this will always download and extract the vsix file